dataactcore/models/lookups.py

```diff
@@ -58,13 +61,13 @@
 PUBLISH_STATUS_DICT = {item.name: item.id for item in PUBLISH_STATUS}
 
 FILE_TYPE = [
-    LookupFileType(1, 'appropriations', '', 'A', 1),
-    LookupFileType(2, 'program_activity', '', 'B', 2),
-    LookupFileType(3, 'award_financial', '', 'C', 3),
-    LookupFileType(4, 'award', '', 'D2', 4),
-    LookupFileType(5, 'award_procurement', '', 'D1', 5),
-    LookupFileType(6, 'awardee_attributes', '', 'E', None),
-    LookupFileType(7, 'sub_award', '', 'F', None)
+    LookupFileType(1, 'appropriations', '', 'A', 1, Appropriation),
+    LookupFileType(2, 'program_activity', '', 'B', 2, ObjectClassProgramActivity),
+    LookupFileType(3, 'award_financial', '', 'C', 3, AwardFinancial),
+    LookupFileType(4, 'award', '', 'D2', 4, AwardFinancialAssistance),
+    LookupFileType(5, 'award_procurement', '', 'D1', 5, AwardProcurement),
+    LookupFileType(6, 'awardee_attributes', '', 'E', None, None),
+    LookupFileType(7, 'sub_award', '', 'F', None, None)
```

Comment:
I like this. The only thing to watch out for is circular dependencies -- so long as we always point lookups to models (and never the reverse) we're fine.